ostd/src/arch/x86/cpu/mod.rs

```diff
@@ -76,6 +77,27 @@ impl UserContext {
     pub fn fp_regs_mut(&mut self) -> &mut FpRegs {
         &mut self.fp_regs
     }
+
+    /// Sets thread-local storage pointer.
+    pub fn set_tls_pointer(&mut self, tls: usize) {
+        self.set_fsbase(tls)
+    }
+
+    /// Gets thread-local storage pointer.
+    pub fn tls_pointer(&self) -> usize {
+        self.fsbase()
+    }
+
+    /// Loads thread-local storage pointer from fsbase register.
+    pub fn load_tls_pointer(&self) {
```

Comment:
This method is confusing. `load_xxx` usually means reading xxx from memory or some other storage medium. But apparently, this means does not load TLS pointer from a source. I suggest calling it `activate_tls_pointer` and revising the Rust doc accordingly.